subsys/logging/Kconfig

```diff
@@ -249,6 +249,23 @@ config LOG_BACKEND_UART_FORMAT_TIMESTAMP
 	default y
 	help
 	  When enabled timestamp is formatted to hh:mm:ss:ms,us.
+	  
+config LOG_FRONTEND
+	bool "Enable GPIO frontend"
+	help
+	  When enabled frontend is using GPIO pins to output logs.
+
+config LOG_FRONTEND_PORT_NUMBER
+	int "Choose port number"
+	depends on LOG_FRONTEND
+	default 1
+	range 0 1
+
+config LOG_FRONTEND_LSB_PIN_NUMBER
+	int "Pin number for least significant bit"
+	depends on LOG_FRONTEND
+	default 1
+	range 0 23
```

Comment:
this shall be dependent on selected port
Port 0 has 32 PINs
Port 1 has 16 PINs